### PR TITLE
47 design connection page

### LINKED ISSUE #47: Design connection page
**Short description:**
Design connection page with the Figma

**Describe the solution you'd like**

### PR DESCRIPTION
**Pull request by TomDUVAL-MAHE**
# Purpose

This pull request brings the overall design of the login page. It must be improved later. Sorry for the size of the commits, which are also poorly linked.

Close #47 

### Previously : 

image

image

### After : 

image

image

image

### CODE REVIEW

File: application/app_code/lib/main.dart [hunk 1]
```diff
@@ -14,6 +14,8 @@ class MyApp extends StatelessWidget {
       title: 'A = L²',
       theme: ThemeData(
         primarySwatch: Colors.blue,
+
+        /// Change to color materialColorBlueForArea()
```
Comment (LucasTesnier) [inline application/app_code/lib/main.dart]: ??

File: application/app_code/lib/pages/signup/signup_state.dart [hunk 2]
```diff
@@ -49,95 +49,123 @@ class SignupPageState extends State<SignupPage> {
     return '';
   }
 
+  Widget displayLogo() {
```
Comment (LucasTesnier) [inline application/app_code/lib/pages/signup/signup_state.dart:53]: Missing doc

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Answer ??? And add documentation, otherwise NTR

File: application/app_code/lib/pages/login/login_state.dart [hunk 3]
```diff
@@ -16,6 +16,8 @@ class LoginPageState extends State<LoginPage> {
   /// password to login with
   String? _password;
 
+  bool _isConnexionWithEmail = false;
```
Comment (LucasTesnier) [inline application/app_code/lib/pages/login/login_state.dart:20]: Missing doc my bad

File: application/app_code/lib/pages/login/login_state.dart [hunk 4]
```diff
@@ -73,6 +75,154 @@ class LoginPageState extends State<LoginPage> {
...
+            });
+          },
+          child: const Text('Login'),
+        ),
+      ),
+      displayResetAndForgotPassword(),
+    ]);
+  }
+
+  /// These function display the button for request a new password
```
Comment (LucasTesnier) [inline application/app_code/lib/pages/login/login_state.dart]: These ?

File: application/app_code/lib/pages/signup/signup_state.dart (hunk 2 above)
Comment (LucasTesnier) [reply application/app_code/lib/pages/signup/signup_state.dart:53]: Up

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Missing doc

**Review by ZQUEMA — APPROVED** [review]

**Review by LucasTesnier — APPROVED** [review]

**Pull request merged**